src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx

```diff
@@ -126,10 +126,12 @@ describe('oscap', () => {
       ...oscapCreateBlueprintRequest,
       name: 'Oscap test',
     };
-
-    expect(receivedRequest).toEqual(expectedRequest);
+    await waitFor(() => {
+      expect(receivedRequest).toEqual(expectedRequest);
+    });
   });
 
+  // happy testing comment, 30000 time out, run 1
```

Comment:
can we remove this comment?:)

Reply:
Will make sure to remove the comment after I'm done with testing! :heart: Still working on this, trying to test a flake that I can't reproduce locally. I've marked the PR as a draft which I should do from the beginning :sweat_smile: